query/stdlib/influxdata/influxdb/to.go

```diff
@@ -172,7 +172,13 @@ func (ToOpSpec) Kind() flux.OperationKind {
 
 // BucketsAccessed returns the buckets accessed by the spec.
 func (o *ToOpSpec) BucketsAccessed() (readBuckets, writeBuckets []platform.BucketFilter) {
-	bf := platform.BucketFilter{Name: &o.Bucket, Organization: &o.Org}
+	bf := platform.BucketFilter{}
+	if o.Bucket != "" {
```

Comment:
no need to change this, but it appears equivalent to the original line.  You avoid 2 assignments in the case where the values are known to be nil

Reply:
Yeah, you are right